src/components/ConfiguredProperty.tsx

```diff
@@ -0,0 +1,20 @@
+import configuredProperties from '@site/config-specs/properties/properties.json';
+
+const CONFIGURED_PROPERTY_NAMES = Object.keys(configuredProperties);
+
+export function isConfiguredProperty(propertyKey: string): boolean {
+    return CONFIGURED_PROPERTY_NAMES.includes(propertyKey);
+}
+
+export function getConfiguredProperty(propertyKey: string): string {
+    return configuredProperties[propertyKey];
+}
+
+export default function ConfiguredProperty({ propertyKey }) {
```

Comment:
not that familiar with React components, but couldn't this also be typed?

```suggestion
export default function ConfiguredProperty({ propertyKey }: { propertyKey: string }): string | null {
```